tools/platforms/TizenPlatform.hx

```diff
@@ -258,31 +239,20 @@ class TizenPlatform extends PlatformTarget
 
 		for (asset in project.assets)
 		{
-			var path = Path.combine(destination + "res/", asset.targetPath);
-
-			System.mkdir(Path.directory(path));
-
-			if (asset.type != AssetType.TEMPLATE)
+			if (asset.targetPath == "/appinfo.json" || asset.targetPath == "appinfo.json")
 			{
-				if (asset.targetPath == "/appinfo.json")
-				{
-					AssetHelper.copyAsset(asset, path, context);
-				}
-				else
-				{
-					// going to root directory now, but should it be a forced "assets" folder later?
-
-					AssetHelper.copyAssetIfNewer(asset, path);
-				}
-			}
-			else
-			{
-				AssetHelper.copyAsset(asset, path, context);
+				asset.type = AssetType.TEMPLATE;
```

Comment:
_appinfo.json_ has been a special case since _TizenPlatform.hx_ was first created, and no one seems to have noticed it not working. Clearly the intent was to process it as a template, which is what I've done here. But... no one seems to have noticed the problem in seven years, so maybe we don't need the special case?

https://github.com/openfl/lime/blob/ecd7876c4b38853022e368415288ec2b93509ed1/tools/platforms/TizenPlatform.hx#L197-L209